packages/terra-demographics-banner/tests/wdio/demographics-banner-spec.js

```diff
@@ -0,0 +1,70 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny', 'large');
+
+describe('Demographics Banner', () => {
+  describe('Blank', () => {
+    beforeEach(() => browser.url('/#/raw/tests/demographics-banner/blank-demographics-banner'));
```

Comment:
nit: each of these beforeEach can be changed to before since we do not need to refresh the page between test steps